cmsimple/classes/Page.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+namespace XH;
+
+/**
+ * Initiates a page object. Its methods act as a wrapper
+ * of the XH/Pages methods for better OOP access
+ *
+ * @category CMSimple_XH
+ * @package  XH
```

Comment:
No need for the `@category` and `@package` tag anymore. We have switched from PEAR CS to PSR-2 a while ago, and the namespace declaration on top of the file is more useful anyway.